packages/core/src/components/forms/_control-group.scss

```diff
@@ -215,6 +215,35 @@ Styleguide components.forms.control-group
   }
 
   /*
+  Responsive control groups
+
+  Add the class `pt-fill` to a control group to make all controls expand equally to fill the
+  available space. Then add the class `pt-fixed` to individual controls to revert them to their
+  original default sizes.
+
+  You can adjust the specific size of a control with the `flex-basis` CSS property.
+
+  Markup:
+  <div class="pt-control-group pt-fill">
+    <div class="pt-input-group">
+      <span class="pt-icon pt-icon-people"></span>
+      <input type="text" class="pt-input" placeholder="Find collaborators..." />
+    </div>
+    <button class="pt-button pt-intent-primary pt-fixed">Add</button>
+  </div>
+
+  Styleguide components.forms.control-group.fill
+  */
+
+  &.pt-fill {
+    display: flex;
```

Comment:
That can be resolved with a separate ticket for 2.0. `pt-fixed` already exists, I'm just reusing it here